src/libbson/tests/json/bson_corpus/timestamp.json

```diff
@@ -18,6 +18,11 @@
             "description": "Timestamp with high-order bit set on both seconds and increment",
             "canonical_bson": "10000000116100FFFFFFFFFFFFFFFF00",
             "canonical_extjson": "{\"a\" : {\"$timestamp\" : {\"t\" : 4294967295, \"i\" :  4294967295} } }"
+        },
+        {
+            "description": "Timestamp with high-order bit set on both seconds and increment (not UINT32_MAX)",
```

Comment:
Corresponds to https://github.com/mongodb/specifications/commit/17134395152c287befe67b928d287f64e2f1f95d for [DRIVERS-1319](https://jira.mongodb.org/browse/DRIVERS-1319). This will let us resolve [CDRIVER-3740](https://jira.mongodb.org/browse/CDRIVER-3740).